db/warehouse/migrate/20250614013242_add_information_date_to_hmis_services.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddInformationDateToHmisServices < ActiveRecord::Migration[7.1]
+  def change
+    update_view 'hmis_services', version: 6, revert_to_version: 5
```

Comment:
We got confirmation that the `InformationDate` change was rolled back. So it is _not_ being added to Services. Sorry we must have missed this somehow.

So this file and associated changes can be removed. Thanks and sorry about the back-and-forth!